sdk/cosmos/azure-spring-data-cosmos/src/test/java/com/azure/spring/data/cosmos/repository/repository/LongIdDomainRepository.java

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.spring.data.cosmos.repository.repository;
+
+import com.azure.spring.data.cosmos.domain.LongIdDomain;
+import com.azure.spring.data.cosmos.repository.CosmosRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface LongIdDomainRepository extends CosmosRepository<LongIdDomain, Long> {
```

Comment:
Please make sure this also works for `ReactiveCosmosRepository`

Reply:
added  ReactiveLongIdDomainPartitionPartitionRepositoryIT, please review

Reply:
Looks good, thanks.